### PR TITLE
ECL-299: Added initial files for account service

### PR DESCRIPTION
**Pull request by nemanjatomanovic**

### CODE REVIEW

**Comment by peter-hazell** [comment]
please can you bring in `ISpecBase` (see economic-crime-levy-returns) into the `it` integration tests, and also change `HealthIntegrationSpec` to match the implementation in returns

**Comment by peter-hazell** [comment]
similar to above comment, can you bring in `SpecBase` into the `test` unit tests and update the hello world controller spec to use the base spec

File: conf/application.conf [hunk 1]
```diff
@@ -79,7 +74,7 @@ metrics {
 # Microservice specific config
 
 auditing {
-  enabled = false
+  enabled = true
```
Comment (peter-hazell) [inline conf/application.conf:77]: leave this disabled locally - it should be disabled in our other services as well locally but don't worry about that we can address in other PRs

Comment (nemanjatomanovic) [reply conf/application.conf]: Done.

File: test-common/uk/gov/hmrc/economiccrimelevyaccount/EclTestData.scala [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  val testInternalId: String               = alphaNumericString
+  val testEclRegistrationReference: String = alphaNumericString
+
+  val minRevenue: Long = 0L
+  val maxRevenue: Long = 99999999999L
+  val minApDays: Int   = 1
+  val maxApDays: Int   = 999
+  val minAmlDays: Int  = 1
+  val maxAmlDays: Int  = 365
+  val daysInYear: Int  = 365
```
Comment (peter-hazell) [inline test-common/uk/gov/hmrc/economiccrimelevyaccount/EclTestData.scala:76]: You don't need these in this repo

File: test-common/uk/gov/hmrc/economiccrimelevyaccount/generators/Generators.scala [hunk 3]
```diff
@@ -0,0 +1,136 @@
+/*
```
Comment (peter-hazell) [inline test-common/uk/gov/hmrc/economiccrimelevyaccount/generators/Generators.scala:1]: You can remove this file from the repo and also remove `Regex` as we don't need these for this backend

Comment (peter-hazell) [inline test-common/uk/gov/hmrc/economiccrimelevyaccount/generators/Generators.scala:1]: You can remove this file from the repo and also remove `Regex` as we don't need these for this backend

File: test-common/uk/gov/hmrc/economiccrimelevyaccount/EclTestData.scala [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+case class EnrolmentsWithEcl(enrolments: Enrolments)
+
+case class EnrolmentsWithoutEcl(enrolments: Enrolments)
+
+final case class EclLiabilityCalculationData(
+  relevantApLength: Int,
+  relevantApRevenue: Long,
+  amlRegulatedActivityLength: Int
+)
```
[the remark this answers is not in the document]
Comment (nemanjatomanovic) [reply test-common/uk/gov/hmrc/economiccrimelevyaccount/EclTestData.scala]: Done

**Review by peter-hazell — APPROVED** [review]

**Pull request merged**